src/main/java/org/apache/cassandra/service/StorageServiceMBean.java

```diff
@@ -808,8 +808,10 @@ default public void setConcurrentCompactors(int value) {
      *            The parent keyspace name
      * @param cfName
      *            The ColumnFamily name where SSTables belong
+     * @param isLoadAndStream
+     *            Whether or not arbitrary SSTables should be loaded (and streamed to the owning nodes)
      */
-    public void loadNewSSTables(String ksName, String cfName);
+    public void loadNewSSTables(String ksName, String cfName, boolean isLoadAndStream);
```

Comment:
Since this is an RPC interface, doesn't it break compatibility with jmx clients that don't expect it?

Reply:
Done. (by keeping both ifaces)